crates/cubecl-spirv/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+edition = "2021"
+license.workspace = true
+name = "cubecl-spirv"
+readme.workspace = true
+version.workspace = true
```

Comment:
I'm not sure how much can actually be shared, but I do think there should be some shared debug representation of the IR that can be pretty printed and used by all compilers. It would make optimizations much easier to debug and check. The raw debug print of operations is very hard to read. Some optimizations can definitely be shared though, and I'd like to try and take the reusable parts of the optimizer I'm writing now and put it in a separate crate. But I think that's for a separate PR.